common/pb/serverinfo_ban.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto2";
+message ServerInfo_Ban {
+    optional string admin_id = 1;
+    optional string admin_name = 2;
+    optional string ban_time = 3;
+    optional string ban_length = 4;
+    optional string ban_reason = 5;
+    optional string visible_reason = 6;
```

Comment:
Could you add some basic doc to these formatted like this: https://github.com/estan/protoc-gen-doc#writing-documentation